scripts/scil_filter_tractogram_anatomically.py

```diff
@@ -83,6 +92,71 @@ def _build_arg_parser():
     return p
 
 
+def binarize_labels(atlas, label_list):
+    #Input atlas must be a numpy array
+    mask = np.zeros(atlas.shape, dtype=np.uint16)
+    for label in label_list:
+        bin = atlas == label
```

Comment:
Shadows built-in name `bin`. Please use a more meaningful name that does not shadow built-in names.

Reply:
Done in dbd0b2a.